### DISCUSSION TITLE
Error running dp_train using dp_train

### DISCUSSION THREAD

**Discussion by k50112113** [deepmodeling/deepmd-kit#99] [Q&A] [answered]
Hi:

I have installed tensorflow r1.8 and DeePMD r0.11 on cluster and I am currently trying to train a model.
It works well when I am testing on home node.
But error below occurs when I was trying to submit jobs using PBS scripts:

File "/usr/local/anaconda/5.2.0/3/lib/python3.6/site-packages/tensorflow/python/pywrap_t
ensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/usr/local/anaconda/5.2.0/3/lib/python3.6/site-packages/tensorflow/python/pywrap_t
ensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/usr/local/anaconda/5.2.0/3/lib/python3.6/site-packages/tensorflow/python/pywrap_t
ensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/usr/local/anaconda/5.2.0/3/lib/python3.6/imp.py", line 243, in load_module
    return load_dynamic(name, filename, file)
  File "/usr/local/anaconda/5.2.0/3/lib/python3.6/imp.py", line 343, in load_dynamic
    return _load(spec)
ImportError: libcuda.so.1: cannot open shared object file: No such file or directory

But I did not build my tensorflow with CUDA support and it seems that it is not using the tensorflow I built (I am installing tensorflow's C++ interface)
It seems that dp_train only works with tensorflow's python interface?

Thanks for your time,
Lee, Shao-Chun

**Comment by jameswind** [comment, marked as answer]
Yes, Shao-Chun, dp_train only requires tensorflow's python interface. Most
likely you've installed a python interface of tensorflow's with a GPU
support. Can you double-check?
If you have available GPU machines, you may change the PBS submission
script, require a single GPU, and try again. If not, you may
install tensorflow's python interface with CPU support only.

Best,
Linfeng

**Comment by k50112113** [comment]
Thanks, I'll reinstall my tensorflow with Python Interface and see if it works.
